web/styles/right_sidebar.css

```diff
@@ -60,8 +60,6 @@ $user_status_emoji_width: 24px;
 
 #buddy-list-users-matching-view,
 #buddy-list-other-users {
-    width: 90%;
-    margin-left: 10px;
```

Comment:
For this commit, the change on the left seems good. But I think the reason we didn't use the full width on the right was to avoid the `\vdots` menu slipping under the hiding Simplebar scrollbar on the right side; so we should check if we're introducing a usability problem there.

Reply:
Confirmed this is a problem; this blocked at least one attempt to just move my mouse and click the `vdots` icon:

![image](https://github.com/user-attachments/assets/712546ac-e1a3-43d8-b51e-72af5fbf2a40)

I don't think the previous `width: 90%` here was a good way to do this answer, but we should leave space such that the scrollbar doesn't overlap the `vdots` menu, like we do in the left sidebar.

@terpimost FYI.

Reply:
Yes...I just tested this after reading your comment, and this is definitely something I overlooked.

Reply:
I added a couple of variables for Simplebar's scroll widths (11px, and 15px on hover--the latter of which is needed here). I bumped things over using `margin-right` on those values as a possible solution.